package.json

```diff
@@ -27,7 +27,8 @@
     "test-firefox": "yarn -s test --firefox",
     "tslint": "tslint -c tslint.json --project ./tsconfig.json",
     "lint": "yarn -s tslint && yarn -s bazel:format-lint",
-    "format:prettier": "prettier --write \"**/*.{js,ts,css,scss,json,md,html}\"",
+    "format:prettier": "prettier --write \"**/*.{js,ts,css,scss,json,md}\" && yarn -s format:prettier:html",
+    "format:prettier:html": "prettier --write \"./src/**/*.html\" --parser angular",
```

Comment:
We now use prettier's angular parser for html files to correctly format the new control-flow syntax.